### PR TITLE
Added learning rate in main_literature.py, in afa_CNN_SVM.py, and afa…

### PR DESCRIPTION
**Pull request by vigimella**
…_MLP_SVM.py. Added new model standard_RBF.py to check its real composition.

### CODE REVIEW

**Comment by giacomo-iadarola** [comment]
The new model standard_RBF and the learning_rate variable in the main_literature.py file refer to 2 different features, you should send me 2 different pull-request instead.
I will immediately accept the commits regarding the standard_RBF model, but the learning_rate into main_literature.py requires more changes to be effective. The variable needs to be also added in the _model_selection function and into the literature_models code too, not only in the parser.

**Comment by Djack1010** [comment]
The new model standard_RBF and the learning_rate variable in the main_literature.py file refer to 2 different features, you should send me 2 different pull-request instead.
I will immediately accept the commits regarding the standard_RBF model, but the learning_rate into main_literature.py requires more changes to be effective. The variable needs to be also added in the _model_selection function and into the literature_models code too, not only in the parser.
